### ISSUE TITLE
Script-loading the KhronosGroup glTF sample AnimatedMorphCube fails

### ISSUE THREAD

**Issue by iambumblehead** [KhronosGroup/UnityGLTF#232]
Script-loading the KhronosGroup glTF sample [AnimatedMorphCube][0] fails with an exception error _Unity 2018.1.5f1_ None of the Khronos sample Animated* glb files loads successfully. 

```
Exception: Dictionary must be an object at: meshes[0].primitives[0].targets[0].NORMAL
GLTF.Extensions.JsonReaderExtensions.ReadAsDictionary[AccessorId] (Newtonsoft.Json.JsonReader reader, System.Func`1 deserializerFunc) (at /Users/bumble/software/UnityGLTF/GLTFSerialization/GLTFSerialization/Extensions/GLTFJsonExtensions.cs:321)
```

related:
 * [Pull request to add animation support][2] was closed and not merged :(
 * [A possibly related link][1] indicates that the KhronosGroup Samples including these Animated samples are of dubious quality.

[0]: https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/AnimatedMorphCube/glTF-Binary/AnimatedMorphCube.glb?raw=true
[1]: https://github.com/KhronosGroup/glTF-Sample-Models/issues/108
[2]: https://github.com/KhronosGroup/UnityGLTF/pull/43

**Comment by blgrossMS** [comment]
See comment in #233 . This seems like a deserialization bug we have.

**Comment by AdamMitchell-ms** [comment]
This will be addressed by pull #363

**Comment by pfcDorn** [comment]
Fixed in latest release
